packages/xo-server/src/api/token.mjs

```diff
@@ -42,6 +42,21 @@ delete_.params = {
 
 // -------------------------------------------------------------------
 
+// TODO: an user should be able to delete its own tokens.
+export async function delete_many({ tokens }) {
```

Comment:
`deleteMany`.

Although, maybe we should merge `delete`, `deleteAll` and `deleteMany` to accept either a single id or a pattern.